admin-cli/executor/client.go

```diff
@@ -65,5 +70,24 @@ func NewClient(writer io.Writer, metaAddrs []string) *Client {
 		Meta:   meta,
 		Nodes:  util.NewPegasusNodeManager(metaAddrs, replicaAddrs),
 		Perf:   aggregate.NewPerfClient(metaAddrs),
+	}, nil
+}
+
+func CloseClient(client *Client) error {
+	var errorStrings []string
+	err := client.Meta.Close()
+	if err != nil {
+		fmt.Printf("Error: failed to close meta session [%s]\n", err)
+		errorStrings = append(errorStrings, err.Error())
```

Comment:
What are the possible errors occurring here ? I think they could just be ignored so we would continue to close other components as belows ?

Reply:
Thinks, I agree and modify code.